nixos/modules/image/gpt.nix

```
@@ -0,0 +1,199 @@
+# This module exposes options to build a disk image with a GUID Partition Table
+# (GPT). It uses systemd-repart to build the image.
+
+{ config, pkgs, lib, utils, ... }:
+
+let
+  cfg = config.image;
+
+  partitionOptions = {
+    options = {
+      storePaths = lib.mkOption {
+        type = with lib.types; listOf path;
+        default = [ ];
+        description = lib.mdDoc "The store paths to inlcude in the partition.";
+      };
+
+      stripStorePaths = lib.mkOption {
```

```suggestion
      stripNixStorePrefix = lib.mkOption {
```

The name is very unclear IMHO.